recipes/pyjc/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: pyjc
+  version: "0.1.0"
+
+source:
+  fn: pyjc-0.1.0.tar.gz
+  url: https://pypi.python.org/packages/af/9d/6a789ddbd67caf159bc548c0e99b10bcf36e7e990d3097dba0b8cb4c62f5/pyjc-0.1.0.tar.gz
+  md5: f0ee545672b71f7a048d588c2bff96e6
```

Comment:
please use `sha256`. You can get it directly from:

https://pypi.org/project/pyjc/#files